pkgs/applications/graphics/dia/default.nix

```diff
@@ -1,50 +1,43 @@
-{stdenv, fetchurl, gtk2, pkgconfig, perl, perlXMLParser, libxml2, gettext
-, python, libxml2Python, docbook5, docbook_xsl, libxslt, intltool, libart_lgpl
-, withGNOME ? false, libgnomeui }:
+{ stdenv, fetchgit, autoconf, automake, libtool, gtk2, pkgconfig, perl,
+perlXMLParser, libxml2, gettext, python, libxml2Python, docbook5, docbook_xsl,
+libxslt, intltool, libart_lgpl, withGNOME ? false, libgnomeui,
+gtk-mac-integration }:
 
 stdenv.mkDerivation rec {
-  name = "dia-${minVer}.3";
-  minVer = "0.97";
+  name = "dia-${version}";
+  version = "2017-06-22";
```

Comment:
I  think it is recommended to write something like `0.97.3.20170622` just in case there is ever a `0.97.4`